keyboards/converter/ps2_usb/keymaps/lbibass/keymap.c

```diff
@@ -0,0 +1,119 @@
+#define MAX 1
+#include QMK_KEYBOARD_H
+#include "ps2.h"
+#define _BL 0
```

Comment:
You never use these, maybe delete them?

Reply:
I do in fact use max, but I'll see about removing the other ones.

Reply:
Nope, they are needed.